packages/mobile/src/app/reducers.ts

```diff
@@ -16,6 +16,8 @@ export interface State {
   lastTimeBackgrounded: number
   sessionId: string
   minVersion: string | null
+  pontoEnabled: boolean
+  kotaniEnabled: boolean
```

Comment:
No need to do anything now, we'll probably store remote feature flags in a dedicated reducer.